sdk/src/wallet/wallet/operations/stronghold.rs

```diff
@@ -64,3 +67,46 @@ impl Wallet {
         }
     }
 }
+
+impl Wallet<StrongholdAdapter> {
```

Comment:
Usability. You can use the enum wrapper `StorageManager` OR the underlying types and both will work.

Reply:
But we don't use it, so who does it help?

Reply:
Users of the lib. These are all publicly accessible.

Reply:
Now I got it, took me some time to understand that this is additional to the SecretManager

Reply:
Do you think the whole file should be removed?

Reply:
No, the old implementation should stay for convenience and because it's used in the bindings
I'm just thinking that anyone who wants to use Stronghold should just use the enum variant and then has access to the partially more convenient functions as well, therefore there is no need to have them here "duplicated"

Reply:
I don't agree that you should be forced to use the enum if you want to access a particular function. The argument for only supporting backwards-compatable fns that will be removed in the future is fair, but for fns that have no plans to remove it doesn't make sense.